ts/BUILD.bazel

```diff
@@ -84,49 +84,47 @@ config_setting(
 config_setting(
     name = "verbose_flag",
     flag_values = {
-        ":verbose": "true"
-    }
+        ":verbose": "true",
+    },
 )
 
 config_setting(
     name = "supports_workers_flag",
     flag_values = {
-        ":supports_workers": "true"
-    }
+        ":supports_workers": "true",
+    },
 )
 
 options(
     name = "options",
-    visibility = ["//visibility:public"],
```

Comment:
note, this is just buidifier formatting, apparently it was previously not buildifier-clean